src/presentational/Proposal.tsx

```diff
@@ -0,0 +1,58 @@
+import {Icon, Text, useTheme} from '@ui-kitten/components';
+import React, {useState} from 'react';
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+import globalStyles, {standardPadding} from 'src/styles';
+import Padder from 'presentational/Padder';
+import {CallInspector, formatCallMeta} from 'src/packages/call_inspector/CallInspector';
+import type {Proposal as ProposalType} from '@polkadot/types/interfaces';
+
+export function Proposal({
+  proposal,
+  accessoryLeft,
+  accessoryRight,
```

Comment:
What does `accessoryLeft` and `accessoryRight` mean in a Proposal?

Reply:
I used the same terminology as [ui-kitten](https://akveo.github.io/react-native-ui-kitten/docs/components/button/api#button) for injecting elements on left and right of components. 

the green parts are accessories
![Simulator Screen Shot - iPhone 12 - 2021-07-09 at 16 00 59](https://user-images.githubusercontent.com/1168983/125089994-28113300-e0cf-11eb-999c-8795d520e93c.png)

Reply:
Yeah I got the idea.. My point was that those names are good for generic components like a `Button`, but `Proposal` is specific to our domain, if you don't know what a `Proposal` looks like it's hard to know what you can pass to those props. Would the `ListItem` component from UI Kitten fit the generic use case?

Reply:
`Proposal` can be expanded on click to show some info underneath. this is the only difference to `ListItem`. I will extract that part as a separate `ProposalInfo` component and then replace Proposal with `ListItem`.

Reply:
88c1763
done ✅